taskcluster/ci/signing/kind.yml

```diff
@@ -0,0 +1,24 @@
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - firefox_tv_taskgraph.signing:transforms
+  - taskgraph.transforms.task:transforms
+
+jobs:
+  push:
+    label: Sign for Github
+    description: ''
+    worker-type: signing
+    run-on-tasks-for: [github-release]
+    dependencies:
+      build: 'Firefox for Fire TV - Release build'
```

Comment:
How should I handle this for the `email` case, where it has multiple dependencies?
I'm worried that if I use the `single_dep` loader for this kind, it will make dependency-configuration too inconsistent between kinds.

I'm indifferent though. If you've considered ^ already and you'd prefer using `single_dep`, that's :+1: and I'll implement it in this PR

Reply:
There are a variety of options for specifying depdencies.

Probably the two most relevant here are:
- `single_dep`: This is used when you want a downstream task for each task in the upstream kind, and wnat to pull data for the downstream task from the upstream task. Stuff downstream of builds is a common example of this; we'll have a variety of build types, and then for each build type, we'll have a series of downstream tasks for each, that depend on the build and have the same platform and stuff.
- [`release-deps`](https://searchfox.org/mozilla-central/source/taskcluster/taskgraph/transforms/release_deps.py): This adds all upstream tasks matching some attributes as dependencies of downstream tasks. This is used for stuff that needs to run after all the platform specific stuff, including things like notifications. (Note: this probably doesn't have the best name, and the m-c one has much more logic than we'd want for our current github projects).